apps/meteor/tests/e2e/message-actions.spec.ts

```diff
@@ -150,5 +150,65 @@ test.describe.serial('message-actions', () => {
 		await poHomeChannel.sidenav.openChat(ADMIN_CREDENTIALS.username);
 		await expect(poHomeChannel.content.lastUserMessage).toContainText(message)
 	})
+
+	test('expect forward text file to channel', async () => {
+		const filename = 'any_file.txt'
+		await poHomeChannel.content.sendFileMessage(filename);
+		await poHomeChannel.content.btnModalConfirm.click();
+		await expect(poHomeChannel.content.lastUserMessage).toContainText(filename)
```

Comment:
There are some missing `;` in this file :grimacing: 
Not really an issue, but maybe it's better to have it and avoid errors in case we enable lint here at some point

Reply:
Thank you for pointing it out! Fixed now!